tools/cloudharness-test/cloudharness_test/apitest_init.py

```diff
@@ -66,3 +71,24 @@ def set(self, case, data, context):
                 case.headers = case.headers or {}
                 case.headers["Authorization"] = f"Bearer {data}"
                 case.headers["Cookie"] = f"kc-access={data}"
+
+    UNSAFE_VALUES = ("%")
```

Comment:
Should it be `("%", )` instead? Right now it will work the same as it is a single char, though for future-proofing it might be better to adjust it